### PR TITLE
Feature/user repository class

### LINKED ISSUE #7: Create user repository class and test
UserRepository:

- [x] Holds onto all of the User objects

- [x] Have a parameter to take in user data

- Have methods to determine:

- [x] Given a user’s ID, what is their user data?

- [x] The average step goal amongst all users

### PR DESCRIPTION
**Pull request by GrantXBeard**
Hello, team! Please review this PR!

What does this PR do?
-added userRepository class and testing file.

Where should your reviewer start?
- Line number one on both files.

What (if any) features are you implementing?
- userRepository class and tests.

Does this PR close any open issues?
- Closes #7 

Thank you!!

### CODE REVIEW

**Review by jordan-sullivan — CHANGES_REQUESTED** [review]
Looks great team! Remove comments and good to go! -J

File: test/UserRepository-test.js [hunk 1]
```diff
@@ -1,8 +1,68 @@
...
-import UserRepository from '../src/UserRepository';
+import { expect } from "chai";
+import UserRepository from "../src/UserRepository";
 
-describe('User Repository', () => {
-  it('should be a function', function () {
-    expect(UserRepository).to.be.a('function');
+describe("User Repository", () => {
+  let userData, userRepository;
+  // console.log("1", userData);
```
Comment (kpn678) [inline test/UserRepository-test.js:6]: Please remove console.log()

**Review by kpn678 — COMMENTED** [review]
Looks good, please remove line 6 on UserRepository-test.

File: test/UserRepository-test.js (hunk 1 above)
Comment (GrantXBeard) [reply test/UserRepository-test.js]: console log removed.

**Review by kpn678 — COMMENTED** [review]
Excellent, the changes look good to me! I think it is ready to be merged if Jordan thinks so, too.

**Pull request merged**